plugins/services/src/js/utils/ServiceConfigDisplayUtil.js

```diff
@@ -42,14 +42,16 @@ const ServiceConfigDisplayUtil = {
     );
   },
 
-  getDisplayValue(value) {
-    // Return the emdash character.
-    if (value == null || value === '') {
-      return String.fromCharCode(8212);
+  getDisplayValue(value, isDisabled = false) {
+    if (!isDisabled && (value == null || value === '')) {
+      return <i>Not Configured</i>;
```

Comment:
I think these should be `em` instead of `i`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em